app/controllers/api_controller.rb

```diff
@@ -120,7 +120,7 @@ def set_access_control_headers
   # not have these. They would instead dealing with the /api/auth
   # mechanism.
   #
-  if Vmdb::Application.config.action_controller.allow_forgery_protection
+  if respond_to?(:verify_authenticity_token)
```

Comment:
Thanks for asking @imtayadeway.

If you do:
`config.action_controller.allow_forgery_protection=true` in your config/environments/*

It doesn't enable the `verify_authenticity_token` before_action.

You must explicitly say `protect_from_forgery` in your controller to have this before_action defined.

My original code removed the `protect_from_forgery` from application_controller.rb and this conditional was failing.
I think this change is more correct because we're not concerned if allow_forgery_protection is enabled in the application, we're want to know if this specific controller knows about this before_action.  I have since put back the `protect_from_forgery` below but this change still makes sense to me.